docs/docs/cmd/todo/task/task-add.md

```diff
@@ -62,9 +78,13 @@ Create a new task with a specific due date
 m365 todo task add --title "New task" --listId "AQMkADlhMTRkOGEzLWQ1M2QtNGVkNS04NjdmLWU0NzJhMjZmZWNmMwAuAAADKvwNgAMNPE_zFNRJXVrU1wEAhHKQZHItDEOVCn8U3xuA2AABmQeVPwAAAA==" --dueDateTime 2023-01-01
 ```
 
-## Response
+Create a new task with categories, a completedDateTime, a startDateTime and a status
 
-### Standard response
+```sh
+m365 todo task add --title "New task" --listName "My task list" --categories "None,Preset24" --completedDateTime 2023-12-01 --startDateTime 2023-12-01 --status "notStarted"
```

Comment:
I would use some different categories in the sample

```suggestion
m365 todo task add --title "New task" --listName "My task list" --categories "Red category,Important" --completedDateTime 2023-12-01 --startDateTime 2023-12-01 --status "notStarted"
```

Additionally, we are defining a `completedDateTime`, but our `status` is `notStarted`. I tested this scenario several times, and it appears that the `completedDateTime` option only works when we set the `status` option to `completed`. So, let's add that info to the `completedDateTime` option description.